Branch-SDK/build.gradle

```diff
@@ -6,6 +6,7 @@ def jarFileName
 dependencies {
     compile fileTree(dir: 'libs', include: '*.jar')
     compile 'com.android.support:support-annotations:22.2.0'
+    compile(name:'answers-shim', ext:'jar')
```

Comment:
Yeah we adjusted the package name to be more consistent with the sdks
On Thu, Jun 9, 2016 at 6:24 PM Sojan P.R. <notifications@github.com> wrote:

> In Branch-SDK/build.gradle
> <https://github.com/BranchMetrics/android-branch-deep-linking/pull/254#discussion_r66549401>
> :
>
> > @@ -6,6 +6,7 @@ def jarFileName
> >  dependencies {
> >      compile fileTree(dir: 'libs', include: '*.jar')
> >      compile 'com.android.support:support-annotations:22.2.0'
> > +    compile(name:'answers-shim', ext:'jar')
>
> @stevewilber <https://github.com/stevewilber> just noticed that there is
> change in package name than from the jar used earlier
> com.crashlytics.sdk.android.answers.shim.AnswersOptionalLogger to
> com.crashlytics.android.answers.shim.AnswersOptionalLogger. Is that
> intentional?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/BranchMetrics/android-branch-deep-linking/pull/254/files/e040bcb6e0b78d552d4f4f53c4e4e91330c93a0d#r66549401>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAdLlZrWtDcXv0e3d8s73h7-cXfquXxDks5qKLysgaJpZM4IxqXX>
> .
>